course4/homework1/vector2d.h

```diff
@@ -4,6 +4,41 @@
 
 struct Vector2d
 {
-	// TODO
+private:
+   double mX, mY;
+
+public:
+
+   Vector2d(const double& x, const double& y)
```

Comment:
There is no value in passing double by const ref. Copying double and copying a pointer is exactly the same operation: 8-byte copy.